purefa/setup.py

```diff
@@ -0,0 +1,60 @@
+from codecs import open  # To use a consistent encoding
```

Comment:
Since this is Python 3 only, `setup.py` isn't needed.

Reply:
Ack